### PR TITLE
Support clearing spans

### LINKED ISSUE #75: How to clear all formatting
Is there a simple way to clear all the formatting?

### PR DESCRIPTION
**Pull request by Goooler**
Closes #75.

### CODE REVIEW

**Comment by Goooler** [comment]
I didn't add `clearSpanStyles(textRange)` and `clearRichSpans(textRange)` as it seems we should use `remove*(textRange)` instead.

File: richeditor-compose/src/commonTest/kotlin/com.mohamedrejeb.richeditor/model/RichTextStateTest.kt [hunk 1]
```diff
@@ -177,6 +178,61 @@ class RichTextStateTest {
...
+        // "some" is the default.
+        assertEquals(defaultSpan, richTextState.currentSpanStyle)
+        richTextState.selection = TextRange(12, 17)
+        // "text" is italic.
+        assertEquals(italicSpan, richTextState.currentSpanStyle)
+
+        // Clear all spans.
+        richTextState.clearRichSpans(TextRange(0, 17))
+
+        assertEquals(defaultSpan, richTextState.currentSpanStyle)
```
Comment (Goooler) [inline richeditor-compose/src/commonTest/kotlin/com.mohamedrejeb.richeditor/model/RichTextStateTest.kt:231]: This failed, looks like a new issue:

```
Expected :SpanStyle(color=Color(0.0, 0.0, 0.0, 0.0, None), brush=null, alpha=NaN, fontSize=Unspecified, fontWeight=null, fontStyle=null, ...

Actual   :SpanStyle(color=Color(0.0, 0.0, 0.0, 0.0, None), brush=null, alpha=NaN, fontSize=Unspecified, fontWeight=null, fontStyle=Italic, ...
```

File: richeditor-compose/src/commonTest/kotlin/com.mohamedrejeb.richeditor/model/RichTextStateTest.kt [hunk 2]
```diff
@@ -177,6 +178,61 @@ class RichTextStateTest {
...
+        // "some" is the default.
+        assertEquals(defaultSpan, richTextState.currentSpanStyle)
+        richTextState.selection = TextRange(12, 17)
+        // "text" is italic.
+        assertEquals(italicSpan, richTextState.currentSpanStyle)
+
+        // Clear all spans.
+        richTextState.clearSpanStyles(TextRange(0, 17))
+
+        assertEquals(defaultSpan, richTextState.currentSpanStyle)
```
Comment (Goooler) [inline richeditor-compose/src/commonTest/kotlin/com.mohamedrejeb.richeditor/model/RichTextStateTest.kt:231]: This failed, looks like a new issue:

```
Expected :SpanStyle(color=Color(0.0, 0.0, 0.0, 0.0, None), brush=null, alpha=NaN, fontSize=Unspecified, fontWeight=null, fontStyle=null, ...

Actual   :SpanStyle(color=Color(0.0, 0.0, 0.0, 0.0, None), brush=null, alpha=NaN, fontSize=Unspecified, fontWeight=null, fontStyle=Italic, ...
```

**Comment by MohamedRejeb** [comment]
Thanks for your contribution! 
I will check this soon, I've been a bit busy lately.

**Pull request merged**